docs/glossary/claimable-balance.mdx

```diff
@@ -72,45 +72,54 @@ const sdk = require("stellar-sdk");
 async function main() {
   let server = new sdk.Server("https://horizon-testnet.stellar.org");
 
-  let A = sdk.Keypair.fromSecret("SAQLZCQA6AYUXK6JSKVPJ2MZ5K5IIABJOEQIG4RVBHX4PG2KMRKWXCHJ");
-  let B = sdk.Keypair.fromPublicKey("GAS4V4O2B7DW5T7IQRPEEVCRXMDZESKISR7DVIGKZQYYV3OSQ5SH5LVP");
+  let A = sdk.Keypair.fromSecret(
+    "SAQLZCQA6AYUXK6JSKVPJ2MZ5K5IIABJOEQIG4RVBHX4PG2KMRKWXCHJ",
+  );
+  let B = sdk.Keypair.fromPublicKey(
+    "GAS4V4O2B7DW5T7IQRPEEVCRXMDZESKISR7DVIGKZQYYV3OSQ5SH5LVP",
+  );
 
   // NOTE: Proper error checks are omitted for brevity; always validate things!
 
   let aAccount = await server.loadAccount(A.publicKey()).catch(function (err) {
-    console.error(`Failed to load ${A.publicKey()}: ${err}`)
-  })
-  if (!aAccount) { return }
+    console.error(`Failed to load ${A.publicKey()}: ${err}`);
+  });
+  if (!aAccount) {
+    return;
+  }
 
   // Create a claimable balance with our two above-described conditions.
-  let soon = Math.ceil((Date.now() / 1000) + 60); // .now() is in ms
+  let soon = Math.ceil(Date.now() / 1000 + 60); // .now() is in ms
```

Comment:
why do linters think they know better than humans

Reply:
/shrug I'm not a linter fan but the only thing I hate worse than linting is configuring linters